bin/kano-feedback-coredump

```diff
@@ -2,34 +2,41 @@
 
 # kano-feedback-coredump
 #
-# Copyright (C) 2014, 2015 Kano Computing Ltd.
-# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
-#
-# This script is called everytime a program segfaults
-# It is setup via sysctl kernel.core_pattern by using a pipe symbol
+# Copyright (C) 2014-2019 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
+# This script is called everytime a program segfaults and runs with sudo.
+# It is setup via sysctl kernel.core_pattern by using a pipe symbol.
+
 
 procname=$1
 killsig=$2
 uid=$3
 gid=$4
 unixtime=$5
 
+latest=10
 core_dump_dir="/var/tmp"
 core_dump_file="$core_dump_dir/core-$procname-$killsig-$uid-$gid-$unixtime.dump"
 
+
 function save_core_dump()
 {
-    # save the coredump coming through stdin to a gzipped file
+    # Save the coredump coming through stdin to a gzipped file.
     cat < /dev/stdin | gzip -c - > $1.gz
 }
 
 function remove_old_dumps()
```

Comment:
Now that we are passing arguments into this function, can we add a description of the arguments that are expected?